src/config/webpack.config.js

```diff
@@ -0,0 +1,133 @@
+const rtl = require('postcss-rtl');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const I18nAggregatorPlugin = require('terra-i18n-plugin');
+const i18nSupportedLocales = require('terra-i18n/lib/i18nSupportedLocales');
+
+module.exports = {
+  entry: {
+    'babel-polyfill': 'babel-polyfill',
+    'terra-site': path.resolve(path.join(__dirname, '..', 'Index')),
+    // 'terra-site': path.resolve(path.join(process.cwd(), 'site', 'Index')),
+  },
+  module: {
+    rules: [{
+      test: /\.(jsx|js)$/,
+      exclude: /node_modules/,
+      use: 'babel-loader',
+    },
+    {
+      test: /\.(scss|css)$/,
+      use: ExtractTextPlugin.extract({
+        fallback: 'style-loader',
+        use: [{
+          loader: 'css-loader',
+          options: {
+            sourceMap: true,
+            importLoaders: 2,
+            localIdentName: '[name]__[local]___[hash:base64:5]',
+          },
+        }, {
+          loader: 'postcss-loader',
+          options: {
+            plugins() {
+              return [
+                Autoprefixer({
+                  browsers: [
+                    'ie >= 10',
+                    'last 2 versions',
+                    'last 2 android versions',
+                    'last 2 and_chr versions',
+                    'iOS >= 10',
+                  ],
+                }),
+                rtl(),
+              ];
+            },
+          },
+        },
+        {
+          loader: 'sass-loader',
+          options: {
+            data: '$bundled-themes: mock, consumer;',
+          },
+        }],
+      }),
+    },
+    {
+      test: /\.md$/,
+      use: 'raw-loader',
+    },
+    {
+      test: /\.(png|svg|jpg|gif)$/,
+      use: 'file-loader',
+    },
+    ],
+  },
+  plugins: [
+    new ExtractTextPlugin('[name]-[hash].css'),
+    new HtmlWebpackPlugin({
+      title: 'Site',
+      template: path.join(__dirname, '..', 'index.html'),
+      chunks: ['babel-polyfill', 'terra-site'],
+    }),
+    new I18nAggregatorPlugin({
+      baseDirectory: process.cwd(),
+      supportedLocales: i18nSupportedLocales,
+    }),
+    new PostCSSAssetsPlugin({
+      test: /\.css$/,
+      log: false,
+      plugins: [
+        PostCSSCustomProperties({ preserve: true }),
+      ],
+    }),
+    new webpack.NamedChunksPlugin(),
+  ],
+  resolve: {
+    extensions: ['.js', '.jsx'],
+    modules: [path.resolve(process.cwd(), 'aggregated-translations'), 'node_modules'],
+
+    // See https://github.com/facebook/react/issues/8026
+    alias: {
+      react: path.resolve(process.cwd(), 'node_modules', 'react'),
+      'react-intl': path.resolve(process.cwd(), 'node_modules', 'react-intl'),
+      'react-dom': path.resolve(process.cwd(), 'node_modules', 'react-dom'),
+    },
+  },
+  output: {
+    filename: '[name].js',
+    path: path.join(process.cwd(), 'dist'),
+  },
+  devtool: 'cheap-source-map',
+  devServer: {
```

Comment:
I found [this](https://webpack.js.org/configuration/dev-server/#devserver) in the webpack documents, and I don't think this configuration is used but could be wrong.

Reply:
Lets look into this post release